omnigibson/robots/a1.py

```diff
@@ -0,0 +1,221 @@
+                Note that _default_joint_pos are hardcoded & precomputed, and thus should not be modified by the user.
+                Set this value instead if you want to initialize the robot with a different rese joint position.
+            obs_modalities (str or list of str): Observation modalities to use for this robot. Default is ["rgb", "proprio"].
+                Valid options are "all", or a list containing any subset of omnigibson.sensors.ALL_SENSOR_MODALITIES.
+                Note: If @sensor_config explicitly specifies `modalities` for a given sensor class, it will
+                    override any values specified from @obs_modalities!
+            proprio_obs (str or list of str): proprioception observation key(s) to use for generating proprioceptive
+                observations. If str, should be exactly "default" -- this results in the default proprioception
+                observations being used, as defined by self.default_proprio_obs. See self._get_proprioception_dict
+                for valid key choices
+            sensor_config (None or dict): nested dictionary mapping sensor class name(s) to specific sensor
+                configurations for this object. This will override any default values specified by this class.
+            grasping_mode (str): One of {"physical", "assisted", "sticky"}.
+                If "physical", no assistive grasping will be applied (relies on contact friction + finger force).
+                If "assisted", will magnetize any object touching and within the gripper's fingers.
+                If "sticky", will magnetize any object touching the gripper's fingers.
+            kwargs (dict): Additional keyword arguments that are used for other super() calls from subclasses, allowing
+                for flexible compositions of various object subclasses (e.g.: Robot is USDObject + ControllableObject).
+        """
+        self._hand_part_names = [11, 12, 13, 14, 21, 22, 31, 32, 41, 42, 51, 52]
+        # Run super init
+        super().__init__(
+            relative_prim_path=relative_prim_path,
+            name=name,
+            scale=scale,
+            visible=visible,
+            fixed_base=fixed_base,
+            visual_only=visual_only,
+            self_collisions=self_collisions,
+            load_config=load_config,
+            abilities=abilities,
+            control_freq=control_freq,
+            controller_config=controller_config,
+            action_type=action_type,
+            action_normalize=action_normalize,
+            reset_joint_pos=reset_joint_pos,
+            obs_modalities=obs_modalities,
+            proprio_obs=proprio_obs,
+            sensor_config=sensor_config,
+            grasping_mode=grasping_mode,
+            grasping_direction="upper",
+            **kwargs,
+        )
+
+    @property
+    def model_name(self):
+        return "a1_inspire"
```

Comment:
this should be auto-generated, so need to specify, unless you specifically wanted that name "a1_inspire"